data/serializer/src/main/java/tech/pegasys/teku/api/migrated/AttestationRewardsData.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.migrated;
+
+import java.util.List;
+import java.util.Objects;
+
+public class AttestationRewardsData {
```

Comment:
This isn't linked up at the moment, or we'd have also broken the `OpenApiIntegrationTest`...